### PR TITLE
fix bugs

### PR DESCRIPTION
**Pull request by farah-elkourdi**
- fix Forget password (#125)
- fix Get/Product/{productID} getting 422 error (#130)

### CODE REVIEW

File: routes/products.js [hunk 1]
```diff
@@ -325,7 +325,7 @@ router.put('/:productId', auth, async (req, res) => {
 router.get('/:productId', auth, async (req, res) => {
   try {
     if (!verify.checkIfValidUUID(req.params.productId)) {
-      res.status(422).json({
+      res.status(404).json({
```
Comment (supersid) [inline routes/products.js:328]: @felkourdi This change is not required as this condition checks if the provided product id is a valid UUID or not. For cases where the product id is not found is handled at line 353

**Review by supersid — COMMENTED** [review]
@felkourdi The change for error code 404 is not required as the condition checks if the provided product id is a valid UUID or not. For cases where the product id is not found is handled at line 353

**Comment by farah-elkourdi** [comment]
Yes, I saw your comment on the issue ticket, I will not merge.

@kavishsanghvi this pull request is reflected on staging without merging, please check Heroku is taking changes from the dev branch, not bugs branch.

**Comment by farah-elkourdi** [comment]
@supersid I only kept fixing Forget password (#125)

**Pull request merged**